lib/stan_math

```diff
@@ -1 +1 @@
-Subproject commit dfbb65161e9ebf703101204f76538109836157f4
+Subproject commit 26b2d80136653467e685c8b54cabcf06b929a5c9
```

Comment:
This needs to be reverted to the current develop version.  Feature pull requests shouldn't update the submodule.